packages/program-boilerplate/src/trigger.ts

```diff
@@ -129,8 +129,7 @@ function handleProgramIntrospection(
   try {
     const newTemplate =
       (handleIntrospection &&
-        (handleIntrospection(template, rules, bodyProgram, tenant) ||
-          handleIntrospection(template, rules))) ||
+        handleIntrospection(template, rules, bodyProgram, tenant)) ||
```

Comment:
Is the other case where it was just calling `handleIntrospection(template, rules)` no longer valid?

Reply:
Hmm I guess this could technically be a breaking change. I removed that because the function signature was changed to have the program and tenant info non-optional (not a breaking change) because that's how the backend schema is written. That means the function can't be called without those arguments so I just deleted the other call to it.

This would only break programs that return `undefined` from introspection when the program/tenant ARE present but return something else if they aren't... I can't imagine that's a valid use case but I guess it could technically be a breaking change. Not sure if it's worth sifting through all the program code to check if any behave like that.

Reply:
Sounds like a strange case, and won't apply to the referral program so I'm ok with this.